django/applications/catmaid/management/commands/catmaid_import_data.py

```diff
@@ -204,7 +204,7 @@ def map_or_create_users(self, obj, import_users, replacement_users,
                     setattr(obj, ref, user)
                 else:
                     raise ValueError("Could not find referenced user " +
-                            "\"{}\" in imported. Try using --map-users or ".format(obj_user_ref_id))
+                            f"\"{obj_user_ref_id}\" in imported. Try using --map-users or ") # XXX is some of this message missing?
```

Comment:
Yes, good catch! I think it should be along the lines of:
```
raise CommandError(f"Could not find referenced user \"{obj_user_ref_id}\"" \
        "in imported data. Try using the--map-users option to map all users " \
        "or --username-mapping=\"import-user=target-user\" for individual " \
        "users. Additionally, --create-unknown-users can be used to create " \
        "missing users, optionally naming them automatically using " \
        "--auto-name-unknown-users.")
```

Reply:
Give it another look

Reply:
Looks good to me now, thanks!